container/src/lib.rs

```diff
@@ -131,7 +163,7 @@ mod tests {
         )?;
 
         let host_mounts_before_run_fail = MountList::new().unwrap();
-        let container = Container::new(test_folder_path).unwrap();
+        let mut container = Container::new(test_folder_path, "test_folder").unwrap();
```

Comment:
Now that this test uses the new state implémentation it tries to create à state file in the default path `/var/run/kaps/containers`. The broblème is that only root users have the right to create files in this path.

I don't see how to solve this problème, have you any advice?

Reply:
Can you try running the tests with `sudo -H -E -u ~/.cargo/bin/cargo` ?

Reply:
I try to run this command locally `sudo -H -E ~/.cargo/bin/cargo test` and get an error:

```
error: could not execute process `rustc -vV` (never executed)

Caused by:
  No such file or directory (os error 2)
```

apparently, I have access to rustc as a non-root user.

Reply:
Can you try again with https://github.com/virt-do/kaps/pull/26 ?

Reply:
It fixes the problème!

Reply:
It fixes the problème!